http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -357,7 +357,7 @@ object Config {
   implicit private val subnetworkLabelValueReader: ValueReader[SubnetworkLabel] = stringValueReader.map(SubnetworkLabel)
   implicit private val diskSizeValueReader: ValueReader[DiskSize] = intValueReader.map(DiskSize)
   implicit private val diskTypeValueReader: ValueReader[DiskType] = stringValueReader.map(s =>
-    DiskType.withNameInsensitiveOption(s).getOrElse(throw new RuntimeException(s"Unable to parse diskType from $s"))
+    DiskType.stringToObject.get(s).getOrElse(throw new RuntimeException(s"Unable to parse diskType from $s"))
```

Comment:
I had originally thought it would be better for users to choose `standard` or `ssd` since they don't need to use the exact keywords that google permits. Also thought this would make more sense in the long term as we start to think about multicloud. Is there a benefit to doing it this way?

Reply:
I changed this in config because in API, we're expecting `pd-standard`, so it's pretty surprising that this string is different between config vs API request...I do want to move away from inferring the string representation of the object from object's name tho (like `Standard`)...